glean_parser/go_server.py

```diff
@@ -58,6 +58,8 @@ def generate_metric_type(metric_type: str) -> str:
         return "string"
     elif metric_type == "boolean":
         return "bool"
+    elif metric_type == "datetime":
+        return "string"
```

Comment:
Why is this a string? How is this going to be used?

Reply:
The generated tables will include a column of type `STRING`.  With the `metrics.yaml` file indicating the type of `datetime` the generated view will prefix the existing `STRING` column name with `raw_` and add a new column of type `TIMESTAMP`using the UDF `mozfun.glean.parse_datetime()` to convert the STRING value to the TIEMSTAMP.

As an example of the expected view see the `metrics.datetime` metrics `moz-fx-data-shared-prod.firefox_desktop.spoc`.

Reply:
would it makes sense for the go type to be `Time` or something, so that the user of the glean_parser doesn't need to know what format string glean expects for the datetime fields?

as a go glean_parser user, if I didn't already know what format string should be, how would I figure that out?

Reply:
Yep - will make the change.

Reply:
+1, this should be a native timestamp type converted to ISO-8601 string before putting in a ping payload. Compare https://github.com/mozilla/glean_parser/blob/bb3beb7258fce567f953d7ec829d31dca488059e/glean_parser/templates/go_server.jinja2#L108